### PR TITLE
New progress guide - 10 follows

### PR DESCRIPTION
**Pull request by mozzius**
- new progress guide: follow 10 people
- new dialog to try and get people to follow more people

https://github.com/user-attachments/assets/f6518685-b3ff-46a0-9e5c-2c9c4997b812

# Test plan

Add yourself to the statsig gate. Reset onboarding state, go through onboarding and select some interests

Observe the new progress guide

Make sure the dialog works on all platforms. Try selecting tabs, searching, following etc

### CODE REVIEW

File: src/state/shell/progress-guide.tsx [hunk 1]
```diff
@@ -167,9 +222,14 @@ export function Provider({children}: React.PropsWithChildren<{}>) {
...
-              ref={guideCompleteToastRef}
-              title={_(msg`Algorithm training complete!`)}
-              subtitle={_(msg`The Discover feed now knows what you like`)}
+              ref={fifthFollowToastRef}
+              title={_(msg`Half way there!`)}
+              subtitle={_(msg`Follow 10 accounts`)}
+            />
+            <ProgressGuideToast
+              ref={tenthFollowToastRef}
+              title={_(msg`Task complete - follow 10 accounts!`)}
```
Comment (surfdude29) [inline src/state/shell/progress-guide.tsx:231]:
```suggestion
              title={_(msg`Task complete – follow 10 accounts!`)}
```
maybe with an en-dash?

Comment (gaearon) [reply src/state/shell/progress-guide.tsx:231]: meh, it's already using a wrong dash for a similar experiment, let's clean these up later

File: src/view/com/posts/PostFeed.tsx [hunk 2]
```diff
@@ -254,7 +254,10 @@ let PostFeed = ({
     }
   }, [pollInterval])
 
-  const progressGuide = useProgressGuide('like-10-and-follow-7')
+  const gate = useGate()
+  const progressGuide = useProgressGuide(
+    gate('new_postonboarding') ? 'follow-10' : 'like-10-and-follow-7',
```
Comment (gaearon) [inline src/view/com/posts/PostFeed.tsx:259]: this would incorrectly expose everyone who looks at a feed to the experiment

**Review by gaearon — APPROVED** [review]

**Comment by knotbin** [comment]
How are these accounts chosen? Is there an algorithm or is it all manual?

**Comment by gaearon** [comment]
these are just top search matches, no manual selection. the matches in search are ranked according to something similar to PageRank

**Pull request merged**